docs/history.rst

```diff
@@ -1,6 +1,36 @@
 Change Log
 ==========
 
+2.2.0
+~~~~~
+* Minimum PROJ version is 6.1.0
+* `pyproj.crs` updates:
+    * Updated CRS repr (issue #264)
+    * Add Datum, CoordinateSystem, CoordinateOperation clases (issue #262)
+    * Added CRS mapping to/from CF 1.8 (pull #244)
+    * Added to_proj4_dict (issue #226)
+    * Provide option to "pretty print" WKT (issue #258)
+    * Better handle CompoundCRS (issue #265)
+    * Add consistency for `is_geographic` and `is_projected` (issue #274)
+    * Added check in CRS.to_epsg for when proj_list is null (issue #257)
+    * Disallow creation of non*CRS (eg pipeline) in CRS class (issue #267)
+    * Added support for ITRF, compound EPSG, and urn projection strings in CRS (pull #289)
+    * Remove unused _CRS.proj_version from pxd file (pull #255)
```

Comment:
Personally, I don't think this is worth mentioning (although I did the PR :-)) 
It more clutters this list. If you keep it, I would say that you actually removed the incorrect CRS.proj_version, as that is the consequence for the user (the user should not need to know about the private _CRS)

Reply:
Since this is your PR, I am fine with removing it if you prefer.